koshmar/src/main/java/org/testmonkeys/koshmar/pageobjects/elements/CheckBox.java

```diff
@@ -0,0 +1,29 @@
+package org.testmonkeys.koshmar.pageobjects.elements;
+
+import org.openqa.selenium.WebElement;
+import org.testmonkeys.koshmar.core.elements.actions.ClickAction;
+
+public class CheckBox extends AbstractComponent {
```

Comment:
please add javadoc for class and members, specifically for check/uncheck functionality, to highlight that it will first verify the state of the chebox to see if it needs clicking or not.